addons/website_event/tests/test_website_event.py

```diff
@@ -29,6 +31,87 @@ def test_website_event_pages_seo(self):
         self.assertEqual(intro_event_menu.view_id.website_meta_title, "Hello, world!")
         self.assertEqual(event.website_meta_title, False)
 
+    def test_01_tickets_questions(self):
```

Comment:
Should we rename to "test_website_event_questions" to match naming of other tests?